runtime/criusupport/criusupport.c

```diff
@@ -136,7 +113,7 @@ getNativeString(J9VMThread *currentThread, j9object_t javaString, char **nativeS
 	mutf8String = vmFuncs->copyStringToUTF8WithMemAlloc(currentThread, javaString, J9_STR_NULL_TERMINATE_RESULT, "", 0, mutf8StringBuf, STRING_BUFFER_SIZE, &mutf8StringSize);
 	if (NULL == mutf8String) {
 		vmFuncs->setNativeOutOfMemoryError(currentThread, 0, 0);
-		*resultType = vm->criuSupportJVMCheckpointFailure;
+		*nlsMsgFormat = "Out of memory when allocating native string during java string conversion";
```

Comment:
If I don't add this line an exception message/stack trace won't be printed. (`Exception in thread "main" java.lang.OutOfMemoryError: native memory exhausted`)

Reply:
thats fine, but it should be an NLS message

Reply:
Should I add a nls module to j9jcl.nls for this? The error message won't be printed since the `Exception in thread "main" java.lang.OutOfMemoryError: native memory exhausted` will override it

Reply:
perhaps I misunterstood. what is the output if you omit `*nlsMsgFormat = ... `

Reply:
It just crashes. If I include it it prints a java exception and stack trace.